.github/workflows/ci.yml

```
@@ -29,10 +32,33 @@ jobs:
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          header: bundle-size
+          path: packages/sury/bundle-size.md
+
+      # Run regardless of the bundle-size gate above: a size regression should
+      # fail the job without also hiding coverage or blocking the artifact.
       - run: pnpm coverage
+        if: ${{ !cancelled() }}
         working-directory: packages/sury
 
       - uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)